contrib/video_server/video_server.py

```diff
@@ -166,13 +176,16 @@ def main():
         videos = videos[1:]
     loop_play = args.loop
     fps=args.fps
+    frame_shape = None
+    if args.shape:
+        img_shape = tuple(args.shape)
```

Comment:
frame shape not getting updated with img_shape

Reply:
fixed. I changed variable names for consistency, and forgot one.